tests/py3/test_geo_task.py

```diff
@@ -3,18 +3,16 @@
 
 """Testing the geometry task."""
 
-
-def test_geo_task(fix_py2env, fix_testDir):
+def test_geo_task(fix_py2env,fix_testDir):
```

Comment:
Looks like this file (and others) is reverting the PEP8-compliant formatting from a recent PR (spaces after commas, #). Could you do an auto-reformat?